tests/test_cli.py

```diff
@@ -52,6 +52,7 @@ def test_cli_single_input(self):
             elif test.inputformat == "tsv":
                 # These test only run on TSV inputs
                 self.run_convert(runner, test)
+                self.run_convert(runner, test, "fhir_json")
```

Comment:
## Added `fhir_json` conversion to CLI test
To do this, I changed `run_convert()`: see comment

Reply:
Yes, if it does not break your PR! Thanks!

Reply:
I did this suggestion via a PR instead:
- https://github.com/mapping-commons/sssom-py/pull/529